### PR TITLE
feat: Add CertificateVerificationRequest.not_after

### PR DESCRIPTION
**Pull request by viccuad**
## Description

Needed by https://github.com/kubewarden/policy-evaluator/pull/208.

## Test

Tested as part of policy-evaluator changes.

### CODE REVIEW

File: src/host_capabilities/crypto.rs [hunk 1]
```diff
@@ -23,8 +23,16 @@ pub enum CertificateEncoding {
 }
 
 /// Verify_cert verifies cert against the passed cert_chain
-pub fn verify_cert(cert: Certificate, cert_chain: Option<Vec<Certificate>>) -> Result<bool> {
-    let req = CertificateVerificationRequest { cert, cert_chain };
+pub fn verify_cert(
```
Comment (flavio) [inline src/host_capabilities/crypto.rs:26]: Please add some inline documentation about:

* `cert_chain`: what happens when it's `None`
* `not_after`: RFC 3339 time format string, what happens when this is `None`

Basically copy the inline docs of the struct you've added inside of `mod.rs`

**Review by flavio — CHANGES_REQUESTED** [review]
LGTM, I just requested one tiny change

File: src/host_capabilities/crypto.rs (hunk 1 above)
Comment (viccuad) [reply src/host_capabilities/crypto.rs:33]: Done in https://github.com/kubewarden/policy-sdk-rust/pull/71/commits/197ec3110e48226165bb7c623f3055b10ef8854b

**Review by jvanz — APPROVED** [review]
LGTM! :D

**Review by flavio — APPROVED** [review]
LGTM, ship it

**Pull request merged**